exercises/binary/src/example.c

```diff
@@ -0,0 +1,23 @@
+int len(char *str)
+{
+   int count = 0;
+   while (*str++) {
+      count++;
+   }
+   return count;
+}
+
+int convert(char *input)
+{
+   int input_len = len(input);
```

Comment:
The Problem is "Write a program that will convert a binary number, represented as a string (e.g. '101010'), to its decimal equivalent using first principles "
                                                                           ^^^^^^^^^^^^
First Principles ==> If your language provides a method in the standard library to perform the conversion, pretend it doesn't exist and implement it yourself.